0006-deck-native-fee.md

```diff
@@ -0,0 +1,83 @@
+# Deck native fee
+
+- Status: proposed
+- Type: new feature
+- Related components: 0001-peerassets-transaction-specification.proto
+- Start Date: 27-08-2017
+- Discussion: (fill me in with link to RFC discussion)
+- author: Peerchemist
+
+## Summary
+
+Deck native fee allows for card transfer fee expressed in the deck's native token. Per-deck fee is set upon deck spawn and becomes the requirement of each valid card transfer on the deck.
+Procedure is the same as if every card transfer on the deck also included a card burn transaction for some amount of cards (tokens) which means that each fee is deducted from the total supply of the card on the deck.
+
+
+## Conventions
+- The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Motivation
+
+There is utility in having a deck native fee, notably as card transfer fees act as deflationary force on the card supply allowing for creation of dynamic economic system revolving around token in question.
+
+## Detailed design
+
+There are three types of PeerAssets card transfer transaction:
+`card_issue` - new cards are being issued by the deck spawner,
+`card_burn` - cards are being destroyed by sending them back to deck spawn address,
+`card_transfer` - cards are being assigned to different address.
+
+Individual card is value transfer between Alice and Bob. For example: `{'sender': 'Alice', 'amount': 20, 'receiver': 'Bob'}`.
```

Comment:
typo

Reply:
Where is the typo?

Reply:
Something seems wrong in this sentence:
> Individual card is value transfer between ...

Shouldn't this be something like:
> Individual card **transfer is a** value transfer between ...

Reply:
+1